core/refresh/RefreshContextView.js

```diff
@@ -18,7 +19,7 @@ import {useEffect} from 'react';
  */
 export const [RefreshContextView, refreshContextView] = hoistCmp.withFactory({
     displayName: 'RefreshContextView',
-    model: uses('RefreshContextModel', {publishMode: ModelPublishMode.LIMITED}),
+    model: uses(RefreshContextModel, {publishMode: ModelPublishMode.LIMITED}),
```

Comment:
This ends up being a nice pickup to be able to leave the strings behind and pass the imported class. 👍